
image_width: 640
image_height: 480
camera_name: camera
camera_matrix:
  rows: 3
  cols: 3
  data: [667.9571949265043, 0.0, 324.74801736378413, 0.0, 666.1694176999051, 266.20393862867456, 0.0, 0.0, 1.0]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.08957167547776716, -0.09172148483360254, 0.004179475878818196, -0.0021714895613765503, -0.48545237474491826]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0]
projection_matrix:
  rows: 3
  cols: 4
  data: [667.9571949265043, 0.0, 324.74801736378413, 0.0, 0.0, 666.1694176999051, 266.20393862867456, 0.0, 0.0, 0.0, 1.0, 0.0]
